Lesson32/Ezgi/my-app/src/shared-components/typography/BodyM/styles.css

```diff
@@ -0,0 +1,9 @@
+p {
```

Comment:
For bodyM, font size should be 12px. The font-size in the example does not match the specified font-size which can be  a bit confusing :) 
<img width="648" alt="Screenshot 2024-10-12 at 18 27 06" src="https://github.com/user-attachments/assets/cddd6903-232d-41a8-9bae-234ee4597dfb">